glide.yaml

```diff
@@ -1,24 +1,23 @@
 package: github.com/aelsabbahy/goss
 import:
+- package: github.com/achanda/go-sysctl
 - package: github.com/aelsabbahy/GOnetstat
+- package: github.com/aelsabbahy/go-ps
 - package: github.com/cheekybits/genny
   subpackages:
   - generic
-- package: github.com/urfave/cli
+- package: github.com/docker/docker
+  subpackages:
+  - pkg/mount
 - package: github.com/fatih/color
-- package: github.com/aelsabbahy/go-ps
-  version: unix_race_condition
```

Comment:
Moving off of this branch will break things. This was added due to the race condition explained here: #267